apps/performance-service/src/controllers/metricsController.test.ts

```diff
@@ -0,0 +1,61 @@
+import { Request, Response } from "express";
+import { queryApi } from "../services/influxDb";
+import {
+  createTestScenarioEvents,
+  expectedTransformedInstitutionData,
+} from "../shared/tests/testData/influx";
+import { getPerformanceRoutingJson } from "./metricsController";
+import { wait } from "../shared/tests/utils";
+
+describe("getPerformanceRoutingJson", () => {
+  it("queries influxdb and transforms the data properly", async () => {
+    const req = {
+      params: {},
+    } as unknown as Request;
+
+    const res = {
+      send: jest.fn(),
+    } as unknown as Response;
+
+    const testInstitutionId = `bank1-${crypto.randomUUID()}`;
+    const testInstitutionId2 = `bank2-${crypto.randomUUID()}`;
+
+    await createTestScenarioEvents(testInstitutionId, testInstitutionId2);
+    await wait(2000);
```

Comment:
I had to add a wait period between writing to influx and reading because it doesn't write fast enough to be available for the read if I don't add a wait.